### ISSUE TITLE
Alias e for editor

### ISSUE THREAD

**Issue by HarleyQuinn-Z6** [ohmyzsh/ohmyzsh#10650]
### Describe the proposed feature.

I would like to open the editor with `e`. For some reason this has been done for the emacs plugin but not for the common aliases.

**Comment by carlosala** [comment]
you can easily add an alias! For example, if you use `vim`:

```zsh
alias e=vim
```

**Comment by HarleyQuinn-Z6** [comment]
Yes I have `alias e=${=EDITOR}`. But it seems like the most common alias to make, and it surprises me that it isn't included, while a less common one as is `alias e=emacs` is included.

**Comment by rwmitchell** [comment]
Is it included somewhere besides the 'emacs' plugin?

**Comment by carlosala** [comment]
> Is it included somewhere besides the 'emacs' plugin?

nop, is an alias for emacs
